.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+If possible, please include the following information when submitting an issue:
+
+1. All Openstack environment variables
```

Comment:
On second thoughts, we won't need these. Ideally, they should declare all the variables from README and `openstack_taster` should work when pointed to any OpenStack deployment.